apps/sms/views/conversation/js/conversation.js

```diff
@@ -739,6 +746,40 @@ var ConversationView = {
     });
   },
 
+  handleComposer: function thui_handleComposer(composer, recipients) {
+
+    function generateMmsAttachmentsForComposer(slides, content) {
+      if (typeof content === 'string') {
+        slides.push({
+          content: new Blob([content], { type: 'text/plain' }),
+        });
+      } else {
+        slides.push({
+          content: content.blob,
+          location: content.name
+        });
+      }
+      return slides;
+    }
+
+    Compose.fromMessage({
+      subject: composer.subject,
+      type: composer.type,
+      body: composer.body,
+      attachments: composer.content
+        .reduce(generateMmsAttachmentsForComposer, [])
```

Comment:
I think it won't be necessary with the new approach of using drafts -- we save and recall attachments directly :)